src/ui/public/agg_types/buckets/date_histogram.js

```diff
@@ -29,7 +29,8 @@ export function AggTypesBucketsDateHistogramProvider(timefilter, config, Private
   function setBounds(agg, force) {
     if (agg.buckets._alreadySet && !force) return;
     agg.buckets._alreadySet = true;
-    agg.buckets.setBounds(agg.fieldIsTimeField() && timefilter.getActiveBounds());
+    const timeRange = timefilter.getActiveBounds() || agg.params.timeRange;
```

Comment:
Should the new parameter `timeRange` be declared in the params array declaration?